app/src/main/java/com/duckduckgo/app/statistics/api/StatisticsRequester.kt

```diff
@@ -110,9 +110,17 @@ class StatisticsRequester(
             .subscribe({
                 Timber.v("App atb refresh succeeded, latest atb is ${it.version}")
                 store.appRetentionAtb = it.version
+                storeUpdateVersionIfPresent(it)
             }, {
                 Timber.v("App atb refresh failed with error ${it.localizedMessage}")
             })
+
+    }
+
+    private fun storeUpdateVersionIfPresent(retrievedAtb: Atb) {
+        if (retrievedAtb.updateVersion != null) {
+            store.atb = Atb(retrievedAtb.updateVersion!!)
```

Comment:
the `!!` shouldn't be necessary (it's a `val` and there's a null check in the same scope)